fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/identifiers/FedoraID.java

```diff
@@ -50,24 +67,39 @@
 
     /**
      * Basic constructor.
-     * @param fullId The full identifier.
+     * @param fullId The full identifier or null if root.
      * @throws IllegalArgumentException If ID does not start with expected prefix.
      */
-    public FedoraID(final String fullId) {
-        if (!fullId.startsWith(FEDORA_ID_PREFIX)) {
-            throw new IllegalArgumentException(String.format("ID must begin with %s", FEDORA_ID_PREFIX));
-        }
-        this.fullId = fullId;
+    private FedoraID(final String fullId) {
```

Comment:
Not really, you always have to pass in the entire ID. Now for a resource the `resourceID == fullID`, but if you are at the TimeMap then you **HAVE** to create the FedoraId with the full id (ie. `object1/fcr:versions`) or else the FedoraId won't know it is a timemap.

Reply:
Okay, I was mainly asking about if it was the fullId because one of the `addTo` methods passes in the resource id, so wanted to double check if the variable name and documentation should declare it as fullId